examples/tutorial/basic-5/tests/basic-5.ts

```diff
@@ -0,0 +1,116 @@
+import * as anchor from "@project-serum/anchor";
+import { Program } from "@project-serum/anchor";
+import { Transaction, TransactionInstruction, TransactionMessage, VersionedTransaction } from "@solana/web3.js";
+import { Basic5 } from "../target/types/basic_5";
+
+describe("basic-5", () => {
+  // Configure the client to use the local cluster.
+  //anchor.setProvider(anchor.AnchorProvider.env());
+  let provider = anchor.AnchorProvider.local("http://127.0.0.1:8899")
+
+  const program = anchor.workspace.Basic5 as Program<Basic5>;
+  const user = anchor.web3.Keypair.generate();
+
+  let [action_state] = anchor.web3.PublicKey.findProgramAddressSync(
+    [anchor.utils.bytes.utf8.encode("action-state"),
+    user.publicKey.toBuffer()
+    ],
+    program.programId);
+
+  before(async () => {
+
+    let res = await provider.connection.requestAirdrop(user.publicKey, 10 * anchor.web3.LAMPORTS_PER_SOL);
+
+    let latestBlockHash = await provider.connection.getLatestBlockhash()
+
+    await provider.connection.confirmTransaction({
+      blockhash: latestBlockHash.blockhash,
+      lastValidBlockHeight: latestBlockHash.lastValidBlockHeight,
+      signature: res,
+    });
+
+  });
+
+  it("basic-5: Robot actions!", async () => {
+
+    // Array of instructions
+    const instructions: TransactionInstruction[] = [
+      // First instruction: set up the Solana accounts to be used
+      program.instruction.create({
```

Comment:
This method is deprecated, use `program.methods.<INSTRUCTION_NAME>().instruction()` instead.

Reply:
Done.